Fix:

error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]

error: implicitly declaring library function 'isspace' with type 'int (int)' [-Werror,-Wimplicit-function-declaration]

error: implicit declaration of function 'panic' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- configure.orig	2020-12-25 18:21:05.000000000 -0600
+++ configure	2020-12-25 18:23:26.000000000 -0600
@@ -3470,8 +3470,8 @@
   for (i = 0; i < 256; i++)
     if (XOR (islower (i), ISLOWER (i))
 	|| toupper (i) != TOUPPER (i))
-      exit(2);
-  exit (0);
+      return 2;
+  return 0;
 }
 _ACEOF
 rm -f conftest$ac_exeext
--- generic/tixGrUtl.c.orig	2004-03-27 20:44:56.000000000 -0600
+++ generic/tixGrUtl.c	2020-12-25 18:28:52.000000000 -0600
@@ -13,6 +13,7 @@
  *
  */
 
+#include <ctype.h>
 #include <tixPort.h>
 #include <tixInt.h>
 #include <tixDef.h>
--- generic/tixImgXpm.c.orig	2008-02-27 22:05:29.000000000 -0600
+++ generic/tixImgXpm.c	2020-12-25 18:38:10.000000000 -0600
@@ -13,6 +13,7 @@
  *
  */
 
+#include <ctype.h>
 #include <tixPort.h>
 #include <tixInt.h>
 #include <tixImgXpm.h>
--- generic/tixPort.h.orig	2020-12-25 18:27:26.000000000 -0600
+++ generic/tixPort.h	2020-12-25 18:36:25.000000000 -0600
@@ -60,6 +60,8 @@
 #   endif
 #endif
 
+EXTERN void panic(char * format, ...);
+
 EXTERN Tcl_HashTable *	TixGetHashTable _ANSI_ARGS_((Tcl_Interp * interp,
 			    char * name, Tcl_InterpDeleteProc *deleteProc,
                             int keyType));
